tsconfig.json

```diff
@@ -2,8 +2,9 @@
   "compilerOptions": {
       "strict": true,
       "esModuleInterop": true,
-      "experimentalDecorators": true
+      "experimentalDecorators": true,
+      "sourceMaps": true,
```

Comment:
This will enable source maps for both development and production mode. It's not good from performance side (increases bundle size) and from security side (gives source maps for everyone who wants to study production code).

If you want to enable source maps for development in all project, you need to create separate tsconfig json for development and setup it's usage